modules/identity/src/Volo.Abp.Identity.Web/Pages/Identity/Users/CreateModal.cshtml.cs

```diff
@@ -28,9 +28,20 @@ public async Task OnGetAsync()
         {
             UserInfo = new UserInfoViewModel();
 
+            var roleDtoList = await _identityRoleAppService.GetAllListAsync();
+
             Roles = ObjectMapper.Map<List<IdentityRoleDto>, AssignedRoleViewModel[]>(
                 await _identityRoleAppService.GetAllListAsync()
             );
+
+            var defaultRoles = roleDtoList.Where(r => r.IsDefault).ToList();
```

Comment:
Add `IsDefault` to `AssignedRoleViewModel` to prevent double collection and `Any` usage.